### PR TITLE
fix docs about release

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: guidelines/releases-guidelines/02-kyma-release-process.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ Ensure that the `prow/RELEASE_VERSION` file from the `test-infra` repository on
    * create a GitHub release and trigger documentation update on the official Kyma website
    * trigger provisioning of the cluster from the created release. Use the cluster to test the release candidate.
 
-   > **CAUTION**: The cluster is automatically generated for you, but removal is a manual action. The release cluster, the IP Addresses, and the DNS records must be deleted manually after tests on the given cluster are done.
+   > **CAUTION**: The cluster is automatically generated for you and will be removed after 7 days.
```
Comment (majakurcius) [inline guidelines/releases-guidelines/02-kyma-release-process.md:178]:
```suggestion
   > **CAUTION**: The cluster is automatically generated for you, and it is automatically removed after 7 days.
```

Comment (dbadura) [reply guidelines/releases-guidelines/02-kyma-release-process.md:178]: Yes :)

Comment (majakurcius) [reply guidelines/releases-guidelines/02-kyma-release-process.md:178]: Will you apply the comment then? :)

Comment (majakurcius) [inline guidelines/releases-guidelines/02-kyma-release-process.md:178]:
```suggestion
   > **CAUTION**: The cluster is automatically generated for you, and it is automatically removed after 7 days.
```

**Pull request closed without merging**